### ISSUE TITLE
[Compatibility Question] Zombie Awareness

### ISSUE THREAD

**Issue by slave2midna** [Laike-Endaril/Dynamic-Stealth#107]
Curiously, was wondering if this mod works well in conjunction with Zombie Awareness, and if there is any recommended configurations to Dynamic Stealth that would enhance the experience? I'm sure it probably needs to be tested, just wanted to throw this out here in case someone else has already worked with these two together and has some tips and suggestions or anything they noticed when the two were running side by side.

**Comment by Laike-Endaril** [comment]
I haven't personally tested the combination, and don't have experience with Zombie Awareness in general, so you likely know better than I do whether ZA is working normally, based on your curseforge post.  Though ofc what really matters in a modpack is whether the pack in general is working how you want, even if any particular mods aren't working as they normally would.

That being said, one particular DS setting to keep in mind for mod conflicts though is the full-bypass filter.  Any entities entered there *should* bypass almost all DS systems (including DS AI, ofc), effectively making those entities act as they would if DS were not installed.  Bit of a last resort, but if you want DS in general and it's causing an issue with particular entities, you can use it.

Also note that if you get a NPE (Null Pointer Exception) related to AI, there is a more specific filter for that in DS which will likely fix the issue without removing DS behavior from the entity (server settings -> AI -> add null checks to AI).  Feel free to ask for help setting that one up if you come across it.

If you want, you can also make a new post on curseforge so it's on the 1st page, and see if you get anyone with experience there.

I'll leave this open for a while in case someone with more info comes across it and feels like sharing.

**Comment by mmmilord** [comment]
I was planning of having both in my pack, but you can only really have one or the other, DS will override ZA's AI, but you can add zombie to the full-bypass filter and kinda have ZA's AI? You won't be able to sneak up on them since DS will be excluded, I guess you could have stealth for other mobs except ones you want ZA's AI for, but that doesn't really make sense overall. It's a huge inconsistency in gameplay. ZA has sound, blood and light sources. That being said DS has it's own sound sense system, it's just not as crazy as ZA's. Those little buggers will find you even if you punch a block with your fist.

If you want to test for yourself, use ZA's 
# Block mining before it breaks cause sound senses to spawn
    B:blockHittingEvent_Active=true # 
to test whether or not the ZA AI is active or not, that way you won't confuse the two AI's sound senses. 

Tl;dr you will have to pick one.

**Comment by xelotex** [comment]
zombie awareness and dynamic stealth are compatible, you just need to go in the config file of zombie awareness and set "SightRange" to 0.
